azurerm/resource_arm_dev_test_lab_schedule_test.go

```diff
@@ -0,0 +1,384 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+)
+
+func TestAccAzureRMDevTestLabSchedule_autoShutdownBasic(t *testing.T) {
+	resourceName := "azurerm_dev_test_schedule.test"
+	ri := tf.AccRandTimeInt()
+	location := testLocation()
+	preConfig := testAccAzureRMDevTestLabSchedule_autoShutdownBasic(ri, location)
+	postConfig := testAccAzureRMDevTestLabSchedule_autoShutdownBasicUpdate(ri, location)
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMDevTestLabScheduleDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: preConfig,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDevTestLabScheduleExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "status", "Disabled"),
+					resource.TestCheckResourceAttr(resourceName, "notification_settings.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "notification_settings.0.status", "Disabled"),
+					resource.TestCheckResourceAttr(resourceName, "daily_recurrence.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "daily_recurrence.0.time", "0100"),
+				),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"task_type"},
```

Comment:
This seems odd, is task type simply not returned by the API?

Reply:
Thanks, I had missed setting task_type in read earlier. now resolved.

Reply:
The Ignore task_type line has now been removed